aether-kernel/aether/kernel/api/utils.py

```diff
@@ -124,7 +136,7 @@ def json_printable(obj):
         return obj
 
 
-custom_jsonpath_wildcard_regex = re.compile('(\$\.)*([a-zA-Z0-9_-]+\.)*?[a-zA-Z0-9_-]+\*')
+custom_jsonpath_wildcard_regex = re.compile('(\$)?(\.)?([a-zA-Z0-9_-]*(\[.*\])*\.)?[a-zA-Z0-9_-]+\*')
```

Comment:
regular expressions are hard to read and understand, if you could write a line with an example of the expected string... I would love it! 
_simple suggestion, no action requested_

Reply:
@obdulia-losantos  Good idea. See new changes for explanations.

Reply:
:heart_eyes: